### DISCUSSION TITLE
Wednesday Assigned Tasks

### DISCUSSION THREAD

**Discussion by HannahStrimpel** [frc5567/2023-Robot#202] [Team Posts]
Has the robot base code been created, so it can be pulled and used?

**Comment by pangoliff** [comment]
I'm working on that right now, sorry for it being a bit late.

**Comment by pangoliff** [comment]
It's up now and you should be able to pull off of my branch, sorry for the delay.

**Comment by HannahStrimpel** [comment]
It's okay. Thanks for letting me know that it is up now.
